bodhi/server/static/js/forms.js

```diff
@@ -119,6 +119,16 @@ Form.prototype.data = function() {
         }
     });
 
+    $(this.idx + " select").each(function() {
+        var value = $(this).val();
+        console.log(value)
```

Comment:
I guess this was used for debug, we should probably remove it ?

Reply:
Correct! removed.

Thanks!